tasks.py

```diff
@@ -65,109 +85,144 @@ def docker_compose(context, command, **kwargs):
     help={
         "force_rm": "Always remove intermediate containers",
         "cache": "Whether to use Docker's cache when building the image (defaults to enabled)",
+        "python-ver": "The version of Python to build the container with (default: 3.7)",
     }
 )
-def build(context, force_rm=False, cache=True):
+def build(context, force_rm=False, cache=True, python_ver=None):
     """Build Nautobot docker image."""
-    print("Building Nautobot .. ")
-    command = f"build --build-arg PYTHON_VER={PYTHON_VER}"
+    if python_ver is None:
+        python_ver = context.nautobot.python_ver
+    else:
+        context.nautobot.python_ver = python_ver
+
+    command = f"build --build-arg PYTHON_VER={python_ver}"
+
     if not cache:
         command += " --no-cache"
     if force_rm:
         command += " --force-rm"
+
+    print(f"Building Nautobot with Python {python_ver}...")
     docker_compose(context, command)
 
 
 # ------------------------------------------------------------------------------
 # START / STOP / DEBUG
 # ------------------------------------------------------------------------------
-@task
+@task()
```

Comment:
Are the parens required? I know it seems minor but I feel like this might result in some confusion.